AK/HashTable.h

```diff
@@ -460,6 +461,22 @@ class HashTable {
         return element;
     }
 
+    static ErrorOr<HashTable<T, TraitsForT, IsOrdered>> of(std::initializer_list<T> values)
```

Comment:
Is there any precedent for `of()`? I coudn't find anything, so probably better to stick to `create()` or JS-style `from()`

Reply:
Oh, JS has `of` _and_ `from`, jfc

Reply:
Yea, `create()` is the defecto standard in C++. We have some from's as well in AK already.